packages/teams-js/.npmignore

```diff
@@ -9,5 +9,6 @@ coverage/**
 CONTRIBUTING.md
 gulpfile.js
 karma.conf.js
+prepNewReadme.js
```

Comment:
Sorry if this is a really naive question, but what is the effect of adding this here? What happens if you don't?

Reply:
Not naive at all. All matching names set in this file will be skipped when npm publish is called on this package.
Once a dev adds teams-js package to their repository (with npm/yarn add teams-js), the files they will see under that node module will exclude those set in the npmignore file.

Reply:
Thanks, that's super helpful!